### PR TITLE
support multiple migration

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: migrator.js [hunk 1]
```diff
@@ -194,22 +197,51 @@ function migratorListing(data, feedstockStatus) {
...
-loadJSON("https://raw.githubusercontent.com/regro/cf-graph3/master/status/archrebuild.json",
-         function(response) {
+    migratorContainer.appendChild(canvas);
+    migratorContainer.appendChild(legend);
+
+    return {canvasId : canvas.id, migratorContainerId: migratorContainer.id}
+}
+
+var migrators = [{name : "archrebuild", description : "ARM/PowerPC Migration Status" },
+{name : "rebuild", description : "OpenSSL Migration Status" }];
```
Comment (CJ-Wright) [inline migrator.js]:
```suggestion
{name : "openssl", description : "OpenSSL Migration Status" }];
```

File: index.html [hunk 2]
```diff
@@ -47,11 +47,9 @@ <h4>Compiler Migration Deemed Sufficiently Complete!</h4>
...
                 Thanks for your patience!</p>
                 <!-- Uncomment the following if you want the migration bar --->
-                <div id="migratorDiv" style="text-align:center;">
-                    <canvas id="migratorCanvas" style="background:white;"></canvas>
-                    <legend for="migratorCanvas"></legend>
-                    <script type="text/javascript" src="migrator.js"></script>
-                </div>
+				<div id="migratorDiv" style="text-align:center;">
+				</div>
+				<script type="text/javascript" src="migrator.js"></script>
```
Comment (scopatz) [inline index.html]: Why is this indented so much and why is the script outside of the div?

Comment (scopatz) [reply index.html]: Also, if you modify the index for testing, you need to modify `template.html` for testing

**Comment by scopatz** [comment]
Thanks @CJ-Wright! The JS looks good but there are a couple of small HTML issues

File: index.html (hunk 2 above)
Comment (CJ-Wright) [reply index.html]: The script is outside the div I think because it is modifying it? I did this with @mariusvniekerk so he might know.

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**